From ba8a9c40d8f8cf621d1b6904a544609313d29a50 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Tue, 17 Jan 2012 19:00:30 +0800
Subject: ENGR00172222:imx6_defconfig: enable ARM errata software workaround
 764369

We need enable the following ARM errata software workaround:

    CONFIG_ARM_ERRATA_764369=y

This errata is TKT078684 in i.MX6Q errata list.

In order to enable this errata, we need cherry-pick the following commit:
ARM: 7091/1: errata: D-cache line maintenance operation by MVA may not succeed:
f64f6df4241093ea928b0a263ec53b93b65efc08

In order to apply the above patch successfully and keep the git history, we also
need cherry-pick the following commit:

ARM: pm: add function to set WFI low-power mode for SMP CPUs
292ec42af7c6361435fe9df50cd59ec76f6741c6

These ERRATAs applied to i.MX6Q(cortex-a9:r2p10 smp)

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index b4c25b2..848e4b4 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Fri Dec 30 13:46:51 2011
+# Tue Jan 17 18:55:30 2012
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -378,6 +378,7 @@ CONFIG_ARM_GIC=y
 # CONFIG_PCI_SYSCALL is not set
 # CONFIG_ARCH_SUPPORTS_MSI is not set
 # CONFIG_PCCARD is not set
+CONFIG_ARM_ERRATA_764369=y
 
 #
 # Kernel Features
-- 
1.7.9.5

